<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Weather Example</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="cityLineEdit">
      <property name="placeholderText">
       <string>Enter city name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="fetchButton">
      <property name="text">
       <string>Fetch Weather</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="weatherLabel">
      <property name="text">
       <string>Weather info will be displayed here</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
